git_machete/client.py

```diff
@@ -915,15 +916,15 @@ def fp_hash(branch_: LocalBranchShortName) -> Optional[FullCommitHash]:
         # in order to render the leading parts of lines properly.
         branch: LocalBranchShortName
         for branch in self.up_branch:
-            upstream = self.up_branch[branch]
+            up_branch = self.up_branch[branch]
```

Comment:
So i changed it because i think that ancestor of the branch != upstream branch

Reply:
Huh TBH I use all the terms interchangably, `parent branch` == `upstream branch` == `up branch`... although `upstream` is somewhat brittle, as it also corresponds to `remote tracking branch` (see e.g. `git show @{upstream}`)

Reply:
https://github.com/VirtusLab/git-machete/issues/582